packages/composables/src/useCartItem.ts

```diff
@@ -76,19 +115,20 @@ export function useCartItem(cartItem: Ref<LineItem>): UseCartItemReturn {
   const isPromotion = computed(() => cartItem.value.type === "promotion");
 
   async function removeItem() {
-    const result = await removeCartItem(cartItem.value.id, apiInstance);
-    // broadcastUpcomingErrors(result);
-    await refreshCart();
+    await removeCartItem(cartItem.value.id, apiInstance);
+    refreshCart();
   }
 
   async function changeItemQuantity(quantity: number): Promise<void> {
-    await changeProductQuantity({
+    changeProductQuantity({
```

Comment:
```suggestion
    await changeProductQuantity({
```
why are we removing that? Then method is complete before the whole action is